scripts/post-release.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -eu
+
+git checkout master
+./scripts/bump-version.sh '' $(date -d "$(echo $RELEASE_VERSION | sed -e 's/^\([0-9]\{2\}\)\.\([0-9]\{1,2\}\)\.[0-9]\+$/20\1-\2-1/') 1 month" +%y.%-m.0.dev0)
+git diff --quiet || git commit -anm 'meta: Bump new development version' && git pull --rebase && git push
```

Comment:
Will we DRY this up somehow for the next repo we integrate?

Reply:
This one only applies to Sentry so no need for that. This is why we introduced the support for `post-release.sh` script in Craft and not made something similar the default.